my-site.rb

```diff
@@ -3,6 +3,27 @@
 class MySite < Sinatra::Base
 
   get "/" do
+    erb :index
+  end
+
+  get "/about" do
+    erb :about
+  end
+
+  get "/blog" do
+    erb :blog
+  end
 
+  get "/projects" do
+    erb :projects
   end
+
+  get "/index" do
```

Comment:
It's not exactly necessary, but I would put this route up with the `/` route, just so both "index" routes are in the same area within your routes file